Model/Indexer/Product/Indexer.php

```diff
@@ -21,7 +21,7 @@
  */
 class Indexer implements IndexerActionInterface, MviewActionInterface
 {
-    const HARD_LIMIT_FOR_PRODUCTS = 10000000;
+    const BATCH_SIZE = 1000;
```

Comment:
How about 200

Reply:
Why? I think that if the batch size is too small, we'll fall into too many connections issue with MySQL

Reply:
You are right. Keep it.

Reply:
Magento uses 1000 internally.